_includes/sections/email-providers.html

```diff
@@ -1,321 +1,164 @@
+    <h5><span class="badge badge-danger">No Address book/calendar encryption</span></h5>
+    <p>Soverin uses standard <a href="https://en.wikipedia.org/wiki/CalDAV">CalDAV</a> and <a href="https://en.wikipedia.org/wiki/CardDAV">CardDAV</a> for calendars and contacts. These protocols do not support <a href="https://en.wikipedia.org/wiki/End-to-end_encryption">E2EE (End-To-End Encryption)</a>. A <a href="/software/calendar-contacts/">standalone option</a> may be more appropiate.</p>
+    <h5><span class="badge badge-info">Extra Functionality</span></h5>
+    <p>Soverin also providers users with space for a personal webpage.</p>
+    </div>
+  </div>
+  <div class="row mb-2">
+    <div class="col-lg-3 col-sm-12 pt-lg-5">
+      <img
+        src="/assets/img/svg/3rd-party/disroot.svg"
+        data-theme-src="/assets/img/svg/3rd-party/disroot-dark.svg"
+        height="70"
+        width="200"
+        class="img-fluid d-block mr-auto ml-auto align-middle"
+        alt="Disroot">
+    </div>
+    <div class="col">
+    <h2>Disroot <span class="badge badge-info">Free</span></h2>
+    <p><strong><a href="https://disroot.org/en/services/email">Disroot</a></strong> offers email amongst <a href="https://disroot.org/en/#services">other services</a>. The service is maintained by volunteers and its community. They have been in operation since <strong>2015</strong>. Disroot is based in <span class="flag-icon flag-icon-nl"></span> Amsterdam. Disroot is free and uses open source software such as Rainloop to provide service. Users support the service through donations and buying extra storage. The mailbox limit is 1 GB, but extra storage can be purchased 0.15€ per GB per month paid yearly.</p>
+    <h5><span class="badge badge-success">Own Domain</span></h5>
+    <p>Disroot lets users use their own domain. They have aliases, however you must <a href="https://howto.disroot.org/en/tutorials/email">manually apply</a> for them.</p>
+    <h5><span class="badge badge-success">Cryptocurrency</span></h5>
+    <p>Disroot accepts PayPal, direct deposit, <strong>Bitcoin</strong>, <strong>Faircoin</strong>, and Patreon when purchasing extra mailbox storage. Donations can be made through Librepay, Flattr, and Monero as well.</p>
+    <h5><span class="badge badge-success">Two Factor Authentication (2FA)</span></h5>
+    <p>Disroot supports two factor authentication for webmail only with <a href="https://en.wikipedia.org/wiki/Time-based_One-time_Password_Algorithm">TOTP</a>. They do not allow security key authentication such as with a <a href="https://en.wikipedia.org/wiki/YubiKey">Yubikey</a> and <a href="https://en.wikipedia.org/wiki/Nitrokey">Nitrokey</a>.</p>
+    <h5><span class="badge badge-success">Encryption at rest</span></h5>
+    <p>Disroot uses full disk encryption.</p>
```

Comment:
For Soverin we said:

> However it doesn't appear to be "zero access", meaning it is technically possible for them to decrypt the data they have

Is this also true here?

```suggestion
    <p>Disroot uses full disk encryption. However, it doesn't appear to be "zero access", meaning it is technically possible for them to decrypt the data they have</p>
```

Otherwise if it is zero-access we should state that as well.

Reply:
It's not zero access no. I am told they might be implementing a system that encrypts with your public key incoming email like Mailbox.org does so that's worth keeping an eye on.